src/main/java/org/usfirst/frc4904/robot/commands/IndexOne.java

```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Intake;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+
+public class IndexOne extends SequentialCommandGroup {
```

Comment:
IMO `IndexOne` is a misleading name, because we don't have that precise of control over our mechanisms.